app/api/admin.py

```diff
@@ -0,0 +1,31 @@
+from fastapi import APIRouter, Request, HTTPException, Depends, Response
+from uuid import UUID
+from ..db import get_database
+from ..models import AccessTokenPayload, AdminMemberUpdate
+from ..auth_helpers import authorize, role_required
+
+router = APIRouter()
+
+@router.put('/member/{id}')
```

Comment:
I think using the uuid validation would be nice her :heavy_check_mark: 
`@route.put('member/{id}, dependencies=[Depends(validate_uuid)]')`